### PR TITLE
Fix notice in debug mode on the dashboard (admin)

### PR DESCRIPTION
**Pull request by aleeks**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | Fix notice in debug mode on the dashboard (admin) |
| Type? | bug fix |
| Category? | BO |
| BC breaks? | no |
| Deprecations? | no |
| How to test? | Go to dashboard in debug mode |

#### Important guidelines
- Make sure [your local branch is up to date](https://help.github.com/articles/syncing-a-fork/) before commiting your changes!
- Your code MUST respect [our Coding Standards](http://doc.prestashop.com/display/PS16/Coding+Standards) (for code written in PHP, JavaScript, HTML/CSS/Smarty/Twig, SQL)!
- Your commit name MUST respect our [naming convention](http://doc.prestashop.com/display/PS16/How+to+write+a+commit+message)!

### CODE REVIEW

**Comment by mickaelandrieu** [comment]
Hi,

good job!

Mickaël

**Review by PrestaEdit — APPROVED** [review]

File: classes/controller/AdminController.php [hunk 1]
```diff
@@ -1989,10 +1989,12 @@ private function getTabs($parentId = 0, $level = 0)
                 unset($tabs[$index]);
             }
 
-            foreach ($tabs[$index]['sub_tabs'] as $sub_tab) {
-                if ((int)$sub_tab['current'] == true) {
-                    $tabs[$index]['current'] = true;
-                    $tabs[$index]['current_level'] = $sub_tab['current_level'];
+            if (!empty($tabs[$index]) && !empty($tabs[$index]['sub_tabs'])) {
```
Comment (thierrymarianne) [inline classes/controller/AdminController.php]: You might prefer the use of `array_key_exists` here instead of `empty`.

**Review by thierrymarianne — CHANGES_REQUESTED** [review]

File: classes/controller/AdminController.php (hunk 1 above)
Comment (mickaelandrieu) [inline classes/controller/AdminController.php]: I was thinking the same thing, but then you will have to do

```php
if (array_key_exists($tabs,$index) && !empty($tabs[$index]) && ...)
```

it's not worth for the same result, isn't it ? :)

File: classes/controller/AdminController.php [hunk 2]
```diff
@@ -1989,10 +1989,12 @@ private function getTabs($parentId = 0, $level = 0)
                 unset($tabs[$index]);
             }
 
-            foreach ($tabs[$index]['sub_tabs'] as $sub_tab) {
-                if ((int)$sub_tab['current'] == true) {
-                    $tabs[$index]['current'] = true;
-                    $tabs[$index]['current_level'] = $sub_tab['current_level'];
+            if (array_key_exists($index, $tabs) && array_key_exists('sub_tabs', $tabs[$index])) {
```
Comment (mickaelandrieu) [inline classes/controller/AdminController.php]: what if ``$tabs[$index]`` is empty ?

**Review by thierrymarianne — APPROVED** [review]

**Pull request merged**